inc/options/class-wpseo-option-titles.php

```diff
@@ -193,6 +193,50 @@ public function get_separator_options() {
 		return $separators;
 	}
 
+	/**
+	 * Get the available separator options.
+	 *
+	 * @return array
```

Comment:
Can you describe the return value?

Reply:
Yep also the summary is a copy-pasta, will update.